src/backend/requirements.txt

```diff
@@ -7,3 +7,5 @@ torch==1.12.1
 fastapi==0.111.0
 uvicorn==0.30.1
 alphashape==1.3.1
+sqlalchemy
+psycopg2-binary
```

Comment:
yes, it was originally psycopg2, the binary version seemed to include things needed from postgres as well, we switched because some people in the team didn't have postgres installed and were having a hard time setting up